curriculum/js0/5.js

```diff
@@ -7,7 +7,13 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a > b && a > c) {
+    return a;
+  }
+  if (b > c ) {
+    return b;
+  }
+  return c;
```

Comment:
Looks good. This is a case where dropping the brackets (optional) can really shorten the code:
```javascript
if (a > b && a > c) return a;
if (b > c ) return b;
return c;
```